test/E2ETests/project.json

```diff
@@ -21,6 +21,14 @@
                 "System.Net.Http": "",
                 "System.Xml": ""
             }
+        },
+        "dnxcore50": {
```

Comment:
Would be okay to run these tests only on CoreCLR? They spawn a server anyway, right?